test/bin/vault/test_usage.py

```diff
@@ -210,4 +210,42 @@ def test_stash_exception_view(self):
             args= parse_args(["archive", "--stash"])
         except:
             success = True
-        self.assertTrue(success)
\ No newline at end of file
+        self.assertTrue(success)
+
+    def test_file_exception_fofn(self):
+        success = False
+        try:
+            args= parse_args(["archive", "--stash", "/file1", "/file2", "--fofn" "/file3"])
+        except:
+            success = True
+        self.assertTrue(success)
+
+        success = False
+        try:
+            args= parse_args(["archive", "--fofn" "/file3", "/file1", "/file2"])
+        except:
+            success = True
+        self.assertTrue(success)
+
+        success = False
+        try:
+            args= parse_args(["keep", "--fofn" "/file3", "/file1", "/file2"])
+        except:
+            success = True
+        self.assertTrue(success)
+
+        success = False
+        try:
+            args= parse_args(["recover", "--fofn" "/file3", "/file1", "/file2"])
+        except:
+            success = True
+        self.assertTrue(success)
+
+        success = False
+        try:
+            args= parse_args(["untrack", "--fofn" "/file3", "/file1", "/file2"])
+        except:
```

Comment:
Ideally, we shouldn't be using generic `except` statements, we should know what is being thrown. We may want to know if it isn't throwing the correct exception, or we may want to interrupt the tests - a generic `except` would catch a `KeyboardInterrupt`